modules/azure-auth-header/azure-auth-header.c

```diff
@@ -54,6 +56,14 @@ _get_rfc1123date(gchar *buf, gsize buf_len)
   return len;
 }
 
+static void
+_get_rfc1123data_or_assert(gchar *date)
+{
+  gsize date_len = 0;
+  date_len = _get_rfc1123date(date, MAX_DATE_LEN);
```

Comment:
With my previous comment, this function extraction/abstraction is not necessary.